tests/unit/helpers.spec.js

```diff
@@ -1,135 +1,132 @@
-(function() {
-  'use strict';
+'use strict';
+const chai = require('chai');
+const helper = require('../../lib/helpers');
+const httpMocks = require('node-mocks-http');
+const spies = require('chai-spies');
 
-  var chai = require('chai'),
-    helper = require('../../lib/helpers'),
-    httpMocks = require('node-mocks-http'),
-    spies = require('chai-spies');
+describe('Helpers test', function() {
+  let expect;
 
-  describe('Helpers test', function() {
-    var expect;
+  beforeEach(function() {
+    expect = chai.expect;
```

Comment:
@akabiru Sure it makes sense to have it as constant since it never changes all through